src/scripts/main.js

```diff
@@ -1 +1,96 @@
 'use strict';
+
+const selectDocument = document.body;
+
+function firstPromise() {
+  const resolver = (resolve, reject) => {
+    selectDocument.addEventListener('click', () => {
+      resolve();
+    });
+
+    setTimeout(() => {
+      if (selectDocument.getElementsByClassName('success')) {
+        reject(false);
+      }
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+}
+
+const promise1 = firstPromise();
+
+promise1
+  .then(clicked)
+  .catch(notClicked);
+
+function clicked() {
+  const newElement = document.createElement('div');
+
+  selectDocument.append(newElement);
+  newElement.classList.add('success');
+  newElement.setAttribute('data-qa', 'notification');
+  newElement.innerHTML += 'First promise was resolved';
+}
```

Comment:
make this one main function and just pass all needed arguments. and reuse it throughout your code